attest/certification.go

```diff
@@ -157,6 +176,65 @@ func (p *CertificationParameters) Verify(opts VerifyOpts) error {
 	return nil
 }
 
+// NewActivateOpts creates options for use in generating an activation challenge for a certified key.
+// The computed hash is the name digest of the public key used to verify the certification of our key.
+func (p *CertificationParameters) NewActivateOpts(verifierPubKey tpm2.Public, ek crypto.PublicKey) (*ActivateOpts, error) {
```

Comment:
This method doesn't seem to use its receiver?

Maybe have this be documentation above the VerifierKeyNameDigest field in ActivateOpts?

```
// Use tpm2.Public.Name() to produce the digest for a provided key. 
```